src/current/_includes/releases/v23.2/v23.2.0-alpha.4.md

```diff
@@ -0,0 +1,197 @@
+## v23.2.0-alpha.4
+
+Release Date: October 23, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-0-alpha-4-general-changes">General changes</h3>
+
+- Updated the `licenses/CCT.txt` file to reflect the latest [Cockroachdb Community License](https://www.cockroachlabs.com/cockroachdb-community-license/). [#112494][#112494]
+
+<h3 id="v23-2-0-alpha-4-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Renamed [cluster settings](../v23.2/cluster-settings.html) related to physical cluster replication for consistency. For example, `bulkio.stream_ingestion.minimum_flush_interval` is now `physical_replication.consumer.minimum_flush_interval`. [#111197][#111197]
+
+<h3 id="v23-2-0-alpha-4-sql-language-changes">SQL language changes</h3>
+
+- [`SHOW SCHEDULES`](../v23.2/show-schedules.html) has two columns that surface the schedule options. These columns have been renamed to align with the documented option names: `on_previous_running` and `on_execution_failure`. [#111759][#111759]
+- Added support for the PLpgSQL `CLOSE` statement, which allows a PLpgSQL routine to close a cursor with the name specified by a cursor variable. [#111330][#111330]
+- When a [`RESTORE`](../v23.2/restore.html) with `remove_regions` is performed, the restore job will now fail if the object contains a [`REGIONAL BY ROW`](../v23.2/multiregion-overview.html#table-locality) table. [#111443][#111443]
```

Comment:
Would it be clearer if the link to "REGIONAL BY ROW" went to this page: https://www.cockroachlabs.com/docs/v23.1/table-localities#regional-by-row-tables?

Reply:
done